src/Symfony/Component/TypeInfo/TypeFactoryTrait.php

```diff
@@ -171,6 +171,10 @@ public static function array(?Type $value = null, ?Type $key = null, bool $asLis
      */
     public static function iterable(?Type $value = null, ?Type $key = null, bool $asList = false): CollectionType
     {
+        if (true === $asList) {
```

Comment:
;)
```suggestion
        if (true === (true === $asList)) {
```

oops sorry ;)
```suggestion
        if ($asList) {
```

Reply:
:laughing: